### PR TITLE
Release/v0.11.3

### PR DESCRIPTION
**Pull request by prashant-shahi**
Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

File: deploy/docker-swarm/clickhouse-setup/docker-compose.yaml [hunk 1]
```diff
@@ -111,7 +111,7 @@ services:
       - clickhouse
 
   otel-collector-metrics:
-    image: signoz/signoz-otel-collector:0.55.3
+    image: signoz/signoz-otel-collector:0.63.0
```
Comment (srikanthccv) [inline deploy/docker-swarm/clickhouse-setup/docker-compose.yaml:114]: When is release published for this?

File: README.md [hunk 2]
```diff
@@ -78,6 +78,12 @@ We support [OpenTelemetry](https://opentelemetry.io) as the library which you ca
 - Python
 - NodeJS
 - Go
+- PHP
+- .NET
+- Ruby on Rails
```
Comment (srikanthccv) [inline README.md:83]:
```suggestion
- Ruby
```

**Review by srikanthccv — APPROVED** [review]

File: deploy/docker-swarm/clickhouse-setup/docker-compose.yaml (hunk 1 above)
Comment (ankitnayan) [reply deploy/docker-swarm/clickhouse-setup/docker-compose.yaml:114]: @prashant-shahi I don't see a release published for this yet

File: README.md (hunk 2 above)
Comment (pranay01) [reply README.md]: Lets keep RoR for now. We have docs only for that

**Review by ankitnayan — APPROVED** [review]

**Pull request merged**